cypress/e2e/home_page.cy.ts

```diff
@@ -0,0 +1,6 @@
+describe('The Home Page', () => {
+    it('successfully loads', () => {
+        cy.visit('/'); // change URL to match your dev URL
+        cy.wait(3000);
```

Comment:
It was for debug only, currently you can drop this line

Reply:
done